### PR TITLE
⭐️GitHub stargazers count

### PR DESCRIPTION
**Pull request by marcpicaud**
Hi there,

I just left NodeSchool Paris where I tried a bunch of workshops. Some of them are really great, some of them... nah.

A GitHub stargazers count in front of each workshop name could add a little feedback about a workshop popularity.

image

What do you think ?

### CODE REVIEW

**Comment by RichardLitt** [comment]
I like this idea! I think it would be useful.

**Comment by finnp** [comment]
I love this. I think all we need to do is make sure that this works with the CI.

**Comment by marcpicaud** [comment]
Thank you guys !

@finnp I assumed - by looking at the history of the deploy script - that there is an env. variable `GITHUB_PERSONAL_ACCESS_TOKEN` set in CodeShip. Actually, if it's not the case, the build step will throw an error.

If we need to change something to make it work in CI, please let me know. I'll be happy to do it !

File: scripts/build.sh [hunk 1]
```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+if [[ -f ../.env ]]
```
Comment (finnp) [inline scripts/build.sh]: If you remove this it should work. There will not be a `.env` file on CI, so this would fail.

**Comment by marcpicaud** [comment]
Hi @finnp and @riyadhalnur, thanks for the feedback ♥ Just removed the check on the .env file.

**Pull request merged**